app/channels/api_channel.rb

```diff
@@ -5,10 +5,11 @@ class ApiChannel < ApplicationCable::Channel
   def subscribed
     @key = APIKey.find_by(key: params[:key])
     if params[:key].present? && @key.present?
-      stream_from 'api_feedback'
-      stream_from 'api_flag_logs'
-      stream_from 'api_deletion_logs'
-      stream_from 'api_statistics'
+      if params[:events].present?
```

Comment:
Copying [my comments in CHQ here](https://chat.stackexchange.com/transcript/message/39121232#39121232):

Here's what I'm seeing - Move the above `stream_from`'s into an else for if params[:events].present?, creating a fallback-ish thing. Then revert changes like https://github.com/Charcoal-SE/metasmoke/pull/229/files#diff-a72e387f7b8fc84d29a55dcb95f18f5aL10

Could have both systems running at the same time. Redis wouldn't care, looks like they're namespaced differently.

Could be a cheap way to reduce/prevent rework on userscripts. Might be good to migrate eventually so we're not maintaining two, but deprecate one and move at our leisure instead of a hard cut